net-p2p/ed2k_hash/ed2k_hash-0.4.0-r3.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="https://downloads.sourceforge.net/ed2k-tools/${P}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="amd64 ppc ~sparc x86"
-IUSE="" # fltk support is broken, bug #359643
+#USE="fltk" # fltk support is broken, bug #359643
```

Comment:
Wow this ebuild is something else. But you should really add `fltk` back and this should be masked via profiles instead.

Although I don't know if it makes sense to keep this package. Drop this commit from this batch and rest looks good. Need to figure out what to do with this package, maybe remove it.